src/index.js

```diff
@@ -39,31 +41,39 @@ if (mainOptions.command === 'start') {
 
   polka({ server })
     .use(serve(publicDir))
-    .get('*', (req, res, next) => readFile(join(publicDir, 'index.html'), 'utf8', (error, body) => {
-      if (error) next(error);
-      else res.end(body);
-    }))
-    .listen(config.port, err => {
+    .get("*", (req, res, next) =>
+      readFile(join(publicDir, "index.html"), "utf8", (error, body) => {
+        if (error) next(error);
+        else res.end(body);
+      })
+    )
+    .listen(config.port, async (err) => {
       if (err) throw err;
-      console.log(`> Running on localhost:${config.port}`);
+      const url = `http://localhost:${config.port}`;
+
+      console.log(`> Running on ${url}`);
+      if (config.open) await open(url);
```

Comment:
why do you need await?

Reply:
Actually, I don't in this very situation. It can be safely removed as we don't use a promise.

This is what [API Docs](https://www.npmjs.com/package/open) says:

> `open(url)` returns a promise for the spawned child process. You would normally not need to use this for anything, but it can be useful if you'd like to attach custom event listeners or perform other operations directly on the spawned process.

Reply:
Done.